src/protocols/multi_party_ecdsa/gg_2020/test.rs

```diff
@@ -69,6 +69,47 @@ fn test_sign_n8_t4_ttag6() {
     let _ = sign(4, 8, 6, vec![0, 1, 2, 4, 6, 7], 0, &[0]);
 }
 
+#[test]
+fn test_keygen_orchestration() {
+    use rand::Rng;
+    let mut rng = rand::thread_rng();
+    let mut share_count_test: u16;
+    let mut threshold_test: u16;
+    for _count in 0..5 {
+        loop {
+            share_count_test = rng.gen::<u16>() % 16;
+            if share_count_test < 3 {
```

Comment:
why?

Reply:
yea... was a little dicey.. I was trying to prevent running tests like 2-of-2 split or 1-of-2 split. Have changed that up. Let me know if you like this version more.